sklearn/linear_model/tests/test_logistic.py

```diff
@@ -138,9 +139,9 @@ def test_logistic_cv_score_does_not_warn_by_default():
     lr = LogisticRegressionCV(cv=2)
     lr.fit(X, Y1)
 
-    with pytest.warns(None) as record:
+    with warnings.catch_warnings():
+        warnings.simplefilter("error")
         lr.score(X, lr.predict(X))
```

Comment:
Thanks @richardt94. The (un)expected warning was a ChangedBehaviorWarning which has been removed in version 0.22. I think we can just remove this whole test actually.

Reply:
Thanks for the review @jeremiedbb. Would you like me to delete the test in this PR or just close it for now?

Reply:
You can do it in this PR